python/tvm/contrib/target/android_nnapi/relayir_to_nnapi_converter/__init__.py

```diff
@@ -0,0 +1,60 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Converts Relay IR subgraph to Android NNAPI source code
+"""
+import tvm
+from .converter import Converter
+
+
+def convert_relayir_to_nnapi(func):
```

Comment:
I won't call this "convert". Following the naming convention of existing BYOC backends, this is the codegen/compiler of NNAPI, so I'll probably just call it `nnapi_compiler`. Also, can you register it like the following?

```python
@tvm._ffi.register_func("relay.ext.nnapicompiler")
def nnapi_compiler(func):
```

Meanwhile, the directory name `relayir_to_nnapi_converter` should also be improved. Maybe just name it `compiler` or `codegen`.

Reply:
Hi, I plan to name it "codegen" if that's OK. (I don't want to collide with the Python built-in function "compile")
As an extra explanation that matters, the external compiler is actually a C++ stub in `src/relay/backend/contrib/android_nnapi/codegen.cc`, and it's that thing that gets registered as `relay.ext.android_nnapi`. 
The existence of the C++ stub is a historical issue, however, it's not a complex file after all, so would you agree if I completely remove the C++ stub and make all of `relay.ext.android_nnapi` pythonic?

Reply:
We actually prefer a reverse way that makes most of the implementations in C++ instead of Python for better performance. However, given that would be a huge effort, it should be fine for this specific backend to be all in Python IMHO.